tensorflow/stream_executor/cuda/cudart_stub.cc

```diff
@@ -131,6 +131,13 @@ extern __host__ __device__ unsigned CUDARTAPI __cudaPushCallConfiguration(
   return func_ptr(gridDim, blockDim, sharedMem, stream);
 }
 
+extern char CUDARTAPI __cudaInitModule(void **fatCubinHandle) {
+  using FuncPtr = cudaError_t(CUDARTAPI *)(void **fatCubinHandle);
+  static auto func_ptr = LoadSymbol<FuncPtr>("__cudaInitModule");
+  if (!func_ptr) return GetSymbolNotFoundError();
```

Comment:
same here, but I'm not sure what char to return if not found. @chsigg

Reply:
0, 1, or π
;-)

It's not documented, but probably Bastiaan has access to the implementation.

Reply:
done. too eager with copy-paste.
0 on failure

Reply:
Thanks @bas-aarts and @chsigg!